components/squad/wikis/starcraft2/squad_custom.lua

```diff
@@ -12,14 +12,12 @@ local Faction = require('Module:Faction')
 local Logic = require('Module:Logic')
```

Comment:
possibly oos: would like to enable squadAuto for sc2

transfer data is getting to a level it can be used on a few team pages

still need to check if we can drop the status storage (extradata) though

possibly as follow up?

Reply:
just checked, currently for some usecases we need some sort of indicator if a player is part of the main roster or not
personally i would like to make it so that if they are not part of the main roster they get something stored in extradata
that way it would only be for the non auto aspect and would also reduce the cases where it would be stored

will add a (temp) tracking category in squad template to check how many cases we have

Reply:
\> 40 teams use it
hence not able to drop it any time soon